lint/hclconvert/convert.go

```diff
@@ -0,0 +1,219 @@
+package hclconvert
+
+import (
+	"strings"
+
+	"github.com/hashicorp/hcl2/hcl"
+	"github.com/hashicorp/hcl2/hcl/hclsyntax"
+	"github.com/zclconf/go-cty/cty"
+	ctyconvert "github.com/zclconf/go-cty/cty/convert"
+	ctyjson "github.com/zclconf/go-cty/cty/json"
+)
+
+// This file is attributed to https://github.com/tmccombs/hcl2json.
+// convertBlock() is customized to create the same data structure as by previous stein version
```

Comment:
[note]
This files is based on https://github.com/tmccombs/hcl2json/blob/8f36bf6db5a1a018fd136f694cd6ce9b6f3291df/convert.go

As described, `convertBlock()` is customized to create the same data structure as by previous stein version.